docs/high-level-overview.md

```diff
@@ -0,0 +1,18 @@
+You will need to create an app on the developer portal and make it return standardized JSON responses upon ONEm requests, that is [user actions](/building/webhooks/). These requests are launched by the ONEm platform on behalf of your users.
```

Comment:
The developer actually only registers an app on the developer portal, so the create an app on the developer portal and make it return standardized JSON responses upon ONEm requests would better fit in the Getting Started or SDK areas from my pov. Since this is high-level overview I'd go for: You will need to locally develop your app and register it on the developer portal in order to receive an API acess key and a callback URL etc`